_posts/static/2018-02-01-installation-wizard.md

```diff
@@ -12,10 +12,10 @@ description: Now with Wizard
 You need several servers to run Flyve MDM:
 
 * a server running Linux, Apache, Mysql/MariaDB and PHP (a LAMP server) for the backend (GLPI and Flyve MDM for GLPI),
-* a server running Mosquitto,
+* a Linux or Debian server running Mosquitto,
```

Comment:
Debian is a distribution of Linux. 

Better to say "A Ubuntu or Debian server running Mosquitto". Maybe enhance this proposal to allow other distros, as long as the auth plugin is available. We should avoid covering how to build Mosquitto and the plugin to keep the documentation simple.

Reply:
Changing ...